<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallMenuWidget</class>
 <widget class="QWidget" name="CallMenuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background: rgb(70, 113, 136);
}</string>
  </property>
  <widget class="QPushButton" name="BUTTON_PHONEBOOK_MENU">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>90</y>
     <width>208</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 2px solid rgb(255, 255, 255);
	background-color: rgba(64, 110, 191, 200);
	color: rgb(200, 200, 180);
}

QPushButton:pressed {
	border: 2px solid rgb(200, 200, 200);
	background-color: rgba(64, 50, 191, 200);
	color: rgb(200, 200, 180);
}</string>
   </property>
   <property name="text">
    <string>PhoneBook</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_KEYPAD_MENU">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>210</y>
     <width>208</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 2px solid rgb(255, 255, 255);
	background-color: rgba(64, 110, 191, 200);
	color: rgb(200, 200, 180);
}

QPushButton:pressed {
	border: 2px solid rgb(200, 200, 200);
	background-color: rgba(64, 50, 191, 200);
	color: rgb(200, 200, 180);
}</string>
   </property>
   <property name="text">
    <string>Kaypad</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_LOG_MENU">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>330</y>
     <width>208</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 2px solid rgb(255, 255, 255);
	background-color: rgba(64, 110, 191, 200);
	color: rgb(200, 200, 180);
}

QPushButton:pressed {
	border: 2px solid rgb(200, 200, 200);
	background-color: rgba(64, 50, 191, 200);
	color: rgb(200, 200, 180);
}</string>
   </property>
   <property name="text">
    <string>Call Log</string>
   </property>
  </widget>
  <widget class="QWidget" name="WIDGET_PHONEBOOK" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>811</width>
     <height>511</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <widget class="QListWidget" name="LISTWIDGET_PHONEBOOK">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>371</width>
      <height>511</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollBar:vertical {
	width: 40px;
}</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <property name="spacing">
     <number>15</number>
    </property>
   </widget>
   <widget class="QListWidget" name="LISTWIDGET_PHONEBOOK_DETAIL">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>441</width>
      <height>511</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="spacing">
     <number>15</number>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="WIDGET_KEYPAD" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>811</width>
     <height>511</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <widget class="QPushButton" name="BUTTON_KEY_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 40px;
	background-color: rgb(229,229,230);
}

QPushButton:pressed {
	border: none;
	border-radius: 40px;
	background-color: rgb(107,117,141);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EDIT_INPUT">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BUTTON_KEY_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 40px;
	background-color: rgb(229,229,230);
}

QPushButton:pressed {
	border: none;
	border-radius: 40px;
	background-color: rgb(107,117,141);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BUTTON_KEY_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 40px;
	background-color: rgb(229,229,230);
}

QPushButton:pressed {
	border: none;
	border-radius: 40px;
	background-color: rgb(107,117,141);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BUTTON_KEY_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 40px;
	background-color: rgb(229,229,230);
}

QPushButton:pressed {
	border: none;
	border-radius: 40px;
	background-color: rgb(107,117,141);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BUTTON_KEY_9">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 40px;
	background-color: rgb(229,229,230);
}

QPushButton:pressed {
	border: none;
	border-radius: 40px;
	background-color: rgb(107,117,141);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BUTTON_KEY_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 40px;
	background-color: rgb(229,229,230);
}

QPushButton:pressed {
	border: none;
	border-radius: 40px;
	background-color: rgb(107,117,141);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BUTTON_DIAL">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Dial</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BUTTON_KEY_BACKSPACE">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>69</width>
      <height>68</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-image: url(:/Keyboard/bs.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BUTTON_KEY_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 40px;
	background-color: rgb(229,229,230);
}

QPushButton:pressed {
	border: none;
	border-radius: 40px;
	background-color: rgb(107,117,141);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BUTTON_KEY_0">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>410</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 40px;
	background-color: rgb(229,229,230);
}

QPushButton:pressed {
	border: none;
	border-radius: 40px;
	background-color: rgb(107,117,141);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BUTTON_KEY_8">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 40px;
	background-color: rgb(229,229,230);
}

QPushButton:pressed {
	border: none;
	border-radius: 40px;
	background-color: rgb(107,117,141);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BUTTON_KEY_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 40px;
	background-color: rgb(229,229,230);
}

QPushButton:pressed {
	border: none;
	border-radius: 40px;
	background-color: rgb(107,117,141);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BUTTON_RE_DIAL">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>230</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Re-Dial</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="WIDGET_LOG" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>811</width>
     <height>511</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <widget class="QListWidget" name="LISTWIDGET_CALL_LOG">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>371</width>
      <height>511</height>
     </rect>
    </property>
    <property name="batchSize">
     <number>100</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="BUTTON_SYNC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>SYNC</string>
   </property>
  </widget>
  <widget class="QLabel" name="LABEL_LOADING">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>WIDGET_KEYPAD</zorder>
  <zorder>BUTTON_PHONEBOOK_MENU</zorder>
  <zorder>BUTTON_KEYPAD_MENU</zorder>
  <zorder>BUTTON_LOG_MENU</zorder>
  <zorder>BUTTON_SYNC</zorder>
  <zorder>WIDGET_PHONEBOOK</zorder>
  <zorder>LABEL_LOADING</zorder>
  <zorder>WIDGET_LOG</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
